plugins/woocommerce/templates/notices/success.php

```diff
@@ -26,7 +26,12 @@
 ?>
 
 <?php foreach ( $notices as $notice ) : ?>
-	<div class="woocommerce-message"<?php echo wc_get_notice_data_attr( $notice ); ?> role="alert">
-		<?php echo wc_kses_notice( $notice['notice'] ); ?>
+	<div class="wc-block-components-notice-banner is-success"<?php echo wc_get_notice_data_attr( $notice ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?> role="alert">
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" aria-hidden="true" focusable="false">
+			<path d="M16.7 7.1l-6.3 8.5-3.3-2.5-.9 1.2 4.5 3.4L17.9 8z"></path>
+		</svg>
+		<div class="wc-block-components-notice-banner__content">
+			<?php echo wc_kses_notice( $notice['notice'] ); ?>
+		</div>
```

Comment:
Can you help walk me thought those changes Niels, we merged existing notices templates? would this apply to all themes or just block themes?

Reply:
Thanks for asking, @senadir. This change affects all themes, both classic and block themes. This change had been requested in https://github.com/woocommerce/woocommerce-blocks/issues/12042 and implemented in https://github.com/woocommerce/woocommerce-blocks/pull/12043. Due to the merge, we've accidentally switched back to the old notices. This PR can be considered an improved version of https://github.com/woocommerce/woocommerce-blocks/pull/12043.